bzt/modules/blazemeter.py

```diff
@@ -157,15 +153,21 @@ def __upload_artifacts(self):
         :return:
         """
         if self.client.token:
-            self.log.info("Uploading all artifacts as jtls_and_more.zip ...")
+            worker_index = os.environ.get('TAURUS_INDEX_ALL', 'no_index')
```

Comment:
And "no index" is bad default. When there's no index, it should be just artifacts.zip. When there is index, it should be artifacts-IDX.zip